### DISCUSSION TITLE
Offline app initialization with shared download folder

### DISCUSSION THREAD

**Discussion by bberberov** [organicmaps/organicmaps#9050] [Ideas]
Is it possible to add completely offline initialization to the app?  I set up the app and downloaded the maps on one device, and then replicated the shared download folder on another device.  But the app on the second device still required me to re-download the initial map.  If the user interface allowed me to select the shared download folder during initialization, I should have been able to start using the app right away, with no downloads.  I hope this may be easy to implement.  Thanks.

**Comment by biodranik** [comment]
There was already a similar issue. The main problem is that app version should match downloaded maps version. Otherwise there will be a lot of surprises.

**Comment by bberberov** [comment]
Sorry, I missed #8025.  I was using the F-Droid version.  I'll have to see what they do differently.

Also, is it possible to mirror the map archive?

**Reply by biodranik** [reply]
You can host a mirror, if you want. Note that it will take a lot of space for more frequent map updates: https://cdn.organicmaps.app/
